generate-all-dockerfile.sh

```diff
@@ -14,48 +16,60 @@ nodeVersions+=' '
 nodeVersions+=$(seq -f "0.12.%g" -s ' ' 0 0)
 
 for repo in $repos; do
+
 	if [ $repo == "armv7hf" ]; then
```

Comment:
you can use a `switch` statement here. Also, no new line after the `for` statement